tests/box/boxview.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* bender-tags: ui, box */
+
+import Box from '/ckeditor5/ui/box/box.js';
+import BoxView from '/ckeditor5/ui/box/boxview.js';
+import Model from '/ckeditor5/ui/model.js';
+
+describe( 'BoxView', () => {
+	let model, view;
+
+	beforeEach( () => {
+		model = new Model( {
+			alignRight: true
+		} );
+
+		view = new BoxView();
+
+		return new Box( model, view ).init();
```

Comment:
Again, I don't think `Box` is necessary here. `BoxView` can be tested without it (https://github.com/ckeditor/ckeditor5-ui-default/issues/26).